lib/perl/Genome/Model/Tools/Bsmap/MethCalcConversionRate.pm

```diff
@@ -0,0 +1,263 @@
+# % Last Change: Thu Jan 15 01:21:18 PM 2015 CST
+package Genome::Model::Tools::Bsmap::MethCalcConversionRate;
+
+use strict;
+use warnings;
+use Genome;
+
+class Genome::Model::Tools::Bsmap::MethCalcConversionRate {
+    is => 'Command',
+    has => [
+        snvs_file => {
+            is => 'String',
+            is_optional => 1,
+            doc => 'Use snvs.hq file to calculate methylation conversion',
+        },
+        model_id => {
```

Comment:
The purpose of this is to give some QC metrics to production on bisulfite data runs. It probably wouldn't be out of place to have this run automatically on every bisulfite model and output into the build dir. I have no idea how to accomplish that though, given that bisulfite is just a refalign model with a specific PP, and that bamQC is disabled for these PPs. If someone more familiar with that corner of the code wants to weigh in, it'd be appreciated.  (@jwalker?)